example/android/app/build.gradle

```diff
@@ -73,11 +78,10 @@ import com.android.build.OutputFile
  */
 
 project.ext.react = [
-    entryFile: "example/index.js",
-    root: "../../../"
+    enableHermes: false,  // clean and rebuild if changing
 ]
 
-apply from: "../../../node_modules/react-native/react.gradle"
+apply from: "../../node_modules/react-native/react.gradle"
```

Comment:
This path is incorrect. The repo is set up in a way that in the root is a library with all required dependencies. In Example directory, there is an example app that is supposed to use the library, react-native, and all other dependencies from the root and not from its own node_modules.

This needs to be reverted.

Reply:
@Trancever  @dratwas said he's fine with creation of 2 separated package.json and simplify this path's.

Reply:
Also it's worth to mention that some of our packages got separate example package.json like `paper`, `viewpager`, `fbads`